docs/cli/how-to-create-model-analysis-cli.md

```diff
@@ -1,157 +1,123 @@
 # How to create a Model Analysis Job in AzureML CLI
 
 This sample will walk you through creating a simple Model analysis dashboard with a single component attatched.
-## Prequisites
-
-- Install the ``` az ml ``` CLI and register your Responsible AI components via [these instructions](https://github.com/Azure/RAI-vNext-Preview/blob/main/docs/Setup.md)
-
-
-
-## Create your first Model Analysis
-### Download and register Model Analysis components for Private Preview
-To install the private preview components for RAI model analysis first [download this zip file and save it in a retrievable location]().
-
-If you don't have it, create a compute cluster called ```cpu-cluster``` by running
-``` Powershell
-az ml compute create -n cpu-cluster --type amlcompute --min-instances 0 --max-instances 10
-```
-Now, create a pipeline job with the following command
-``` Powershell
-az ml job create --file pipeline.yml
-```
-
-Then, register your 'InitiateModelAnalysis.YAML' component and your 'GenerateExplanations' components from that directory. 
-```YAML
-az ml component create –file <InitiateModelAnalysis.yaml>
-az ml component create –file <GenerateExplanations.yaml>
-```
-
-### Step 2: Provide inputs to your Initiate Model Analysis Component
-- After downloading and registering your model analysis components, you will want to navigate to the folder that contains your components.
-- Open the "Initiate_Model_Analysis.YAML" file
 
+## Prequisites
 
-The .YAML config below will initiate your model analysis. There are a few parameters that will need to be ready in advance.
+- Install the ``` az ml ``` CLI and register your Responsible AI components via [these instructions](https://github.com/Azure/RAI-vNext-Preview/blob/main/docs/Setup.md). You will need to remember your responses to the questions posed by the `generate_registration_files.py` script - specifically, the version number specified.
```

Comment:
"You will need to remember your responses to the questions posed by the `generate_registration_files.py` script - specifically, the version number specified." Is very flowery, maybe something more concise?

"Make sure to write down the version number specified when you run the `generate_registration_files.py` script."

Reply:
Done